xbmc/network/NetworkServices.cpp

```diff
@@ -755,8 +756,7 @@ bool CNetworkServices::StopEventServer(bool bWait, bool promptuser)
     if (server->GetNumberOfClients() > 0)
     {
       bool cancelled = false;
-      if (!CGUIDialogYesNo::ShowAndGetInput(13140, 13141, 13142, 20022,
-                                            -1, -1, cancelled, 10000)
+      if (!CGUIDialogYesNo::ShowAndGetInput(13140, 13141, cancelled, -1, -1, 10000)
```

Comment:
this has to be. I suppose you hadn't seen my edited comment on the commit I reviewed yesterday.
```
if (!CGUIDialogYesNo::ShowAndGetInput(13140, 13141, cancelled, "", "", 10000)
```